api/sensuclient.go

```diff
@@ -0,0 +1,84 @@
+package api
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/golang/glog"
+	"net"
+	"time"
+)
+
+const (
+	defaultSensuHost = "sensu.nais:3030"
+	stopCharacter    = "\n"
+)
+
+type message struct {
+	Name        string   `json:"name"`
+	MessageType string   `json:"type"`
+	Handlers    []string `json:"handlers"`
+	Output      string   `json:"output"`
+}
+
+func unixTimeInNano() int64 {
+	return time.Now().UnixNano()
+}
+
+func GenerateDeployMessage(deploymentRequest *NaisDeploymentRequest, clusterName *string) ([]byte, error) {
```

Comment:
could this be "private"

Reply:
Could be, but then I can't test it directly.

Reply:
ok